io.asgardeo.tomcat.oidc.sample/src/main/resources/configuration-catalog.md

```diff
@@ -191,3 +204,67 @@ the page that the user would always be redirected to, after successful authentic
 ```
 homePage=home.jsp
 ```
+
+### Post Logout Redirect URI
+
+**Property Name:** `postLogoutRedirectURI`
+
+**Description:** This is the URI to which the user will be returned to upon successfully signing out from the OpenID Provider. If not configured, the callbackURL will be used instead.
+
+
+**Sample:**
+
+```
+postLogoutRedirectURI=http://localhost:8080/oidc-sample-app/oauth2client
+```
+
+### HTTP Connect Timeout
+
+**Property Name:** `httpConnectTimeout`
+
+**Description:** This parameter denotes the timeout in milliseconds for establishing the connection to the OpenID Provider. 2000 ms (2 seconds) by default.
+
+**Sample:**
+
+```
+httpConnectTimeout=2000
+```
+
+### HTTP Read Timeout
+
+**Property Name:** `httpReadTimeout`
+
+**Description:** This parameter denotes the timeout in milliseconds for reading the data received from the OpenID Provider. 2000 ms (2 seconds) by default.
+
+**Sample:**
+
+```
+httpReadTimeout=2000
+```
+
+### HTTP Size Limit
+
+**Property Name:** `httpSizeLimit`
+
+**Description:** This parameter denotes the HTTP entity size limit in bytes. 51200 bytes (50 KBytes) by default.
+
+**Sample:**
+
+```
+httpSizeLimit=51200
+```
+
+### State
+
+**Property Name:** `state`
+
+**Description:** This parameter may denote an opaque value which can be used to maintain the state between the request and the callback. This can be used to prevent cross-site request forgery (CSRF) attacks. Null by default.
```

Comment:
This is not null by default. If the state param is not provided, we would be setting a random UUID
https://github.com/asgardeo/asgardeo-java-oidc-sdk/blob/master/io.asgardeo.java.oidc.sdk/src/main/java/io/asgardeo/java/oidc/sdk/request/OIDCRequestBuilder.java#L154-L167

Reply:
Resolved via [95d692a](https://github.com/asgardeo/asgardeo-tomcat-oidc-agent/pull/42/commits/95d692ada2c4e0ea35f5fe4ed659f7b5bacac5b1)